.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 !.github/actions/**/node_modules
 .vscode
 dist
+!/test-packages/test-services/openapi/**/dist
```

Comment:
I'll ignore this change and assume all the files under the dist folder will be removed.

Reply:
I try to make the test self-contained again. I hope this is stable....